tests/database/queries/activities.test.js

```diff
@@ -1,91 +1,110 @@
 const test = require('tape');
+const pg = require('pg');
+const { getConfig } = require('../../../config');
+const { refresh: refreshDB, empty: emptyDB } = require('../../../db/scripts');
 const activities = require('../../../react-backend/database/queries/activities');
 const activitiesForToday = require('../../../react-backend/database/queries/activities_today');
 const deleteActivity = require('../../../react-backend/database/queries/activity_delete');
 const insertActivity = require('../../../react-backend/database/queries/activity_add');
 const updateActivity = require('../../../react-backend/database/queries/activity_update');
 
-const rebuild = require('../../../react-backend/database/database_rebuild');
-const clean = require('../../../react-backend/database/database_clean');
+const config = getConfig(process.env.NODE_ENV);
 
 test('Tests activities responds with the correct data', async (t) => {
-  await rebuild();
+  await refreshDB();
 
-  const activityArray = await activities(2);
+  const client = new pg.Client(config.psql);
+  await client.connect();
+
+  const activityArray = await activities(client, 1);
 
   t.equal(activityArray.length, 5, 'Activities gets all activities');
   t.equal(Object.keys(activityArray[0]).length, 9, 'Activity has 9 fields');
 
   try {
-    await activities();
+    await activities(client);
     t.fail('Worked without a cb id');
   } catch (e) {
     t.pass('Activities errors when no cb id given');
   }
 
-  await clean();
+  await emptyDB();
 
-  const activityArrayEmpty = await activities(2);
+  const activityArrayEmpty = await activities(client, 1);
 
   t.deepEqual(activityArrayEmpty, [], 'No error if activities is called on empty table');
+
+  await client.end()
   t.end();
 });
 
 test('Tests activitiesForToday responds with activities by day', async (t) => {
-  await rebuild();
+  await refreshDB();
+
+  const client = new pg.Client(config.psql);
+  await client.connect();
+
+  const activitiesMonday = await activitiesForToday(client, 1, 'monday');
```

Comment:
Okay, I'm going to stop marking these, as we don't actually need to import client 🙈

Reply:
Ignore 😊